packaging/images/Containerfile.rpm-builder

```diff
@@ -1,5 +1,5 @@
 ARG TAG
-FROM registry.ci.openshift.org/ocp-multi/builder:$TAG
+FROM registry.ci.openshift.org/ocp/builder:$TAG
```

Comment:
Is this multiarch too?

Reply:
yes,  validated with :
```
oc image info registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15 --filter-by-os linux/arm64
```

Reply:
looks like `rhel-9-golang-1.20-openshift-4.15` tag doesnt with arm, so we must use `rhel-9-golang-1.20-builder-multi-openshift-4.16` which is the same  image but  mutli arch.